UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Startup.cs

```diff
@@ -47,6 +47,8 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<FileShareServiceConfiguration>(configuration.GetSection("FileShareService"));
             services.Configure<RadioNavigationalWarningsContextConfiguration>(configuration.GetSection("RadioNavigationalWarningsContext"));
             services.Configure<CacheConfiguration>(configuration.GetSection("CacheConfiguration"));
+            services.Configure<BannerNotificationConfiguration>(configuration.GetSection("BannerNotificationConfiguration"));
+         
             services.Configure<AzureAdB2C>(configuration.GetSection("AzureAdB2C"));
 
             services.Configure<AzureAdB2C>(configuration.GetSection("AzureAdB2C"));
```

Comment:
Just noticed this, please remove this duplicate line of code:
services.Configure<AzureAdB2C>(configuration.GetSection("AzureAdB2C"));

Reply:
done required changes.